packages/ipfs-daemon/package.json

```diff
@@ -32,21 +32,20 @@
   "dependencies": {
     "@ceramicnetwork/common": "^1.9.0-rc.0",
     "@ceramicnetwork/ipfs-topology": "^1.1.17-rc.0",
-    "aws-sdk": "^2.902.0",
-    "blockcodec-to-ipld-format": "^1.0.0",
-    "dag-jose": "^0.3.0",
-    "datastore-fs": "^5.0.1",
-    "datastore-level": "^6.0.1",
-    "datastore-s3": "^6.0.0",
+    "aws-sdk": "^2.1034.0",
+    "dag-jose": "^1.0.0",
+    "datastore-fs": "^6.0.1",
+    "datastore-level": "^7.0.1",
+    "datastore-s3": "^8.0.0",
```

Comment:
Ooh, this makes me nervous. Are you sure the major bump would not result in changed behaviour? Do you think their changelog is safe enough?

Reply:
For `datastore-fs` and `datastore-level` the breaking change is `deep imports/requires are no longer possible`. 
For `datastore-s3` the two breaking changes are `only uses named exports` and `uses new interface-datastore types`. The new interface doesn't change much from what I can see. But I also am uncomfortable as we don't have any tests and I only did a few manual tests. I can change this back and do all of them or only `datastore-s3` in the future.

Reply:
Okay, then we'll see how it behaves when deployed.